rollup/ccc/libzkp/Cargo.toml

```diff
@@ -23,7 +23,7 @@ poseidon = { git = "https://github.com/scroll-tech/poseidon.git", branch = "main
 bls12_381 = { git = "https://github.com/scroll-tech/bls12_381", branch = "feat/impl_scalar_field" }
 
 [dependencies]
-prover = { git = "https://github.com/scroll-tech/zkevm-circuits.git", tag = "v0.11.4", default-features = false, features = ["parallel_syn", "scroll", "strict-ccc"] }
+prover = { git = "https://github.com/scroll-tech/zkevm-circuits.git", tag = "v0.11.7", default-features = false, features = ["parallel_syn", "scroll", "strict-ccc"] }
```

Comment:
maybe 0.12.0?

Reply:
the ccc version is in another version series. this pr will upgrade it from `v0.11.7` to `v0.11.8`.